.circleci/src/jobs/sentry_android.yml

```diff
@@ -0,0 +1,21 @@
+# Upload the Android sourcemaps to sentry
+docker:
+  - image: cimg/node:18.14.1
+resource_class: small
+shell: /bin/bash -eo pipefail
+steps:
+  - skip_job
+  - checkout
+  - prepare_workspace
+  - restore_environment_variables
+  # A react-native app reports its version as: application_id@version+build_number
+  - run:
+      name: 'Sentry Upload: integreat'
+      command: tools/sentry-release tuerantuer.app.integreat "${NEW_VERSION_NAME}" ~/attached_workspace/sourcemaps/integreat --version-code "${NEW_VERSION_CODE}"
```

Comment:
I hardcoded all the build config names, bundle ids and application ids, not 100% convinced of it but I think it makes sense. Otherwise we always have to restore the whole yarn cache to evaluate it from the build config and I thought that is not worth it.